docs/imviz/displayimages.rst

```diff
@@ -145,6 +145,36 @@ You can :ref:`import <imviz-import-regions-api>` and :ref:`export <imviz_export_
 There are options available in the :guilabel:`Layer` tab under the |icon-settings-sliders| icon
 to make subsets visible or invisible, to change their color, and to change their opacity.
 
+.. _imviz_defining_spatial_regions_single_pixel:
+
+Single-Pixel Selection
+----------------------
+
+This tool allows the user to create a single-pixel spatial region
+in an image viewer. Activate this tool and then left-click to create
+the new region. Click again to move it to a new location under the cursor.
+Users can hold down the alt key (Alt key on Windows, Option key on Mac)
+while clicking on a pixel to create a new region at that point instead
+of moving it. Users can also utilize the different subset modes that are
+explained in the :ref:`Spatial Regions <imviz_defining_spatial_regions>`
+section above.
```

Comment:
```suggestion
This tool allows the user to create a single-pixel spatial region
in an image viewer. Activate this tool and then left-click to create
the new region. Click again to move the region to a new location under
the cursor. Holding down the alt key (Alt key on Windows, Option key
on Mac) while clicking pixels creates a new region at each point instead
of moving the previously created region. You can also use the subset
modes that are explained in the 
:ref:`Spatial Regions <imviz_defining_spatial_regions>`
section above in the same way you would with the other subset selection
tools.
```

Reply:
I think this improves the clarity a bit, and we generally address "you" rather than "the user" (right? 🤔 ).

Reply:
Should I also apply similar change to https://github.com/spacetelescope/jdaviz/blob/main/docs/cubeviz/displaycubes.rst ? I copied a lot of that text from Cubeviz.

Reply:
OK, I applied your suggested text to both Imviz and Cubeviz docs. But the glue capitalization, I am not sure... maybe future PR? 😬